e2e-test/cypress/integration/project_edit_spec.js

```diff
@@ -0,0 +1,60 @@
+describe("Project Edit", function() {
+  let projects = undefined;
+
+  before(() => {
+    cy.login();
+    cy.fetchProjects().then(p => (projects = p));
+  });
+
+  beforeEach(function() {
+    cy.login();
+    cy.visit(`/projects`);
+  });
+
+  it("Editing the title is possible", function() {
```

Comment:
add some comments to let the reader know what's happening. 
e.g. "getting value of text field" for line 23, "resetting the title" for line 25